crates/pop-cli/src/commands/test/execute_block.rs

```diff
@@ -186,12 +186,14 @@ mod tests {
 		cli::MockCli,
 		common::{
 			runtime::{get_mock_runtime, Feature},
-			try_runtime::{get_try_state_items, source_try_runtime_binary, DEFAULT_LIVE_NODE_URL},
+			try_runtime::{get_try_state_items, source_try_runtime_binary},
 		},
 	};
 	use console::style;
 	use pop_common::Profile;
 
+	const LOCAL_NODE_URL: &str = "ws://localhost:9944";
```

Comment:
Here we use a constant. I'd say we can do this across all tests :ok_hand:

Reply:
https://github.com/r0gue-io/pop-cli/pull/582/commits/cc619af503c84a725f7bd79e54bf345c70c610aa